Formula/bucklespring.rb

```diff
@@ -0,0 +1,25 @@
+class Bucklespring < Formula
+  desc "Nostalgia bucklespring keyboard sound"
+  homepage "https://github.com/zevv/bucklespring"
+  url "https://github.com/zevv/bucklespring/archive/v1.4.0.tar.gz"
+  sha256 "b38b2c0ec905573a243cb9db543f8eeb92e4d040025ef338d8bcfcfd5ccd5348"
+  license "GPL-2.0-only"
+
+  depends_on "make" => :build
+  depends_on "alure"
+  depends_on "pkg-config"
+
+  def install
+    system "
+      cp /usr/local/lib/pkgconfig/alure.pc mac/lib/pkgconfig/ ;
+      sed -i '' 's/-Wall -Werror/-Wall/' Makefile ;
+      PATH_AUDIO=#{prefix}/wav make
+    "
```

Comment:
```suggestion
    inreplace "Makefile", "-Wall -Werror", "-Wall"
    ENV["PATH_AUDIO"] = prefix/"wav"
    system "make"
    (lib/"pkgconfig").install Dir["mac/lib/pkgconfig/*"]
```
Needing to do an `inreplace` (or `sed`) in the `Makefile` isn't good for new formulae, though. This should be fixed upstream first. And installing the `pkgconfig` files should be in `make install`.

Reply:
I'll see if this can be upstreamed.